UndertaleModLib/Models/UndertaleFont.cs

```diff
@@ -76,8 +87,33 @@ public void Unserialize(UndertaleReader reader)
                 SourceY = reader.ReadUInt16();
                 SourceWidth = reader.ReadUInt16();
                 SourceHeight = reader.ReadUInt16();
-                Shift = reader.ReadUInt16();
-                Offset = reader.ReadUInt32();
+                Shift = reader.ReadInt16();
+                if (reader.undertaleData.GeneralInfo?.Major >= 2 || (reader.undertaleData.GeneralInfo?.Major == 1 && reader.undertaleData.GeneralInfo?.Build == 9999))
+                {
+                    Offset = reader.ReadInt16();
+                    Kerning = reader.ReadUndertaleObject<UndertaleSimpleListShort<GlyphKerning>>();
+                } else
+                {
+                    Offset = reader.ReadInt32(); // Maybe? I don't really know, but this definitely used to work
```

Comment:
Maybe it was two bytes of data and two bytes of padding, or maybe the list always existed but was empty?

Reply:
I guess maybe it can assume there's always a list? No real harm in doing so.